### PR TITLE
Completed javascript-modal

### PR DESCRIPTION
**Pull request by Johnny-Baltazar**
javascript-modal

### CODE REVIEW

File: javascript-modal/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div>
+      <button class="modal-button" type="button">Open Modal</button>
+    </div>
+  </div>
+  <div>
+    <dialog class="modal-box">
```
Comment (Zhiwutian) [inline javascript-modal/index.html:17]: While this works, try completing the functionality without the `dialog` element using a standard div

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
functionality is approved! But I would like you to complete it without the `dialog` tag

**Review by Zhiwutian — APPROVED** [review]
Very nice!

**Comment by Johnny-Baltazar** [comment]
javascript-modal

**Pull request merged**